packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -382,18 +380,8 @@ const ComboBox = React.forwardRef((props: ComboBoxProps, ref) => {
     }
   };
 
-  const enabled = useFeatureFlag('enable-v11-release');
-
   const showWarning = !invalid && warn;
-  const className = cx(
-    `${prefix}--combo-box`,
-    [enabled ? null : containerClassName],
-    {
-      [`${prefix}--list-box--up`]: direction === 'top',
-      [`${prefix}--combo-box--warning`]: showWarning,
-      [`${prefix}--combo-box--readonly`]: readOnly,
-    }
-  );
```

Comment:
I don't think all of this should be removed. It's just the `containerClassName` that's behind the v11 feature flag. This will solve the issue with `direction` not being used.

Reply:
Thanks for spotting! I pushed a commit with partial revert.